eng/Versions.props

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MajorVersion>6</MajorVersion>
+    <MajorVersion>7</MajorVersion>
```

Comment:
@ryalanms , looks like we are reusing this variable for fetching package here: pkg/windowsdesktop/bundle/Microsoft.WindowsDesktop.App.Bundle.bundleproj. which , for me, seems  not right and causing build failure because package doesn't yet exist. Can you take a look please?

Reply:
@mmitche , Branding on Desktop repo is going to be blocked until we have a package `VS.Redist.Common.NetCore.%(InsertionName).%(Arch).$(MajorVersion).$(MinorVersion)` published.

Reply:
I would retry tomorrow after more repos have had successful builds.

Reply:
This is the most recent version that was produced by the runtime: 

```xml
<Package Id="VS.Redist.Common.NetCore.SharedFramework.x64.7.0" Version="7.0.0-alpha.1.21417.28" NonShipping="true"/>
```

This is the error building locally: 

D:\windowsdesktop\pkg\windowsdesktop\bundle\Microsoft.WindowsDesktop.App.Bundle.bundleproj : error NU1101: Unable to find package VS.Redist.Common.NetCore.SharedFramework.x64.7.0. No packages exist with this id in source(s): dotnet-eng, dotnet-public, dotnet-tools, dotnet5, dotnet5-transport, dotnet6, dotnet6-transport [D:\windowsdesktop\.dotnet\sdk\6.0.100-rc.1.21379.2\NuGet.targets]

With these additional feeds in nuget.config:

```xml
  <add key="dotnet7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json" />
    <add key="dotnet7-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7-transport/nuget/v3/index.json" />
```

D:\windowsdesktop\pkg\windowsdesktop\bundle\Microsoft.WindowsDesktop.App.Bundle.bundleproj : error NU1102: Unable to find package VS.Redist.Common.NetCore.SharedFramework.x64.7.0 with version (= 6.0.0-rc.1.21417.1) [D:\windowsdesktop\.dotnet\sdk\6.0.100-rc.1.21379.2\NuGet.targets]
D:\windowsdesktop\pkg\windowsdesktop\bundle\Microsoft.WindowsDesktop.App.Bundle.bundleproj : error NU1102:   - Found 1 version(s) in dotnet7-transport [ Nearest version: 7.0.0-alpha.1.21417.28 ] [D:\windowsdesktop\.dotnet\sdk\6.0.100-rc.1.21379.2\NuGet.targets]

I think WindowsDesktop needs the new feeds and the version in versions.props needs to be updated.